access/slack/config.go

```diff
@@ -61,6 +64,13 @@ func LoadConfig(filepath string) (*Config, error) {
 	if err := t.Unmarshal(conf); err != nil {
 		return nil, trace.Wrap(err)
 	}
+	if strings.HasPrefix(conf.Slack.Token, "/") {
+		tokenBytes, err := ioutil.ReadFile(conf.Slack.Token)
```

Comment:
Is there a max token length or anything that we should be concerned about?

Reply:
The length of a token is considered further in `CheckAndSetDefaults`

Reply:
OK, so we check for token absence - but what if someone points this at a 10GB file?

Update: I just checked mainline Teleport, and we don't enforce a bound there so this is probably OK. (see $TELEPORT_REPO/lib/utils/token.go:29-40)

Reply:
Sure. Don't think we need to check token length because it's something that admin provides, not an attacker.